### DISCUSSION TITLE
Foreign keys/cascading deletes

### DISCUSSION THREAD

**Discussion by JohnnyJayJay** [cozodb/cozo#56] [Q&A] [answered]
Hi,

is it possible to have cascading deletes, similar to foreign key relations and `ON DELETE CASCADE` in SQL? If not, how do you model relationships like that?

**Comment by zh217** [comment, marked as answer]
We mainly focus on ad-hoc joins, so adding formal foreign keys is not a priority at the moment. But you can achieve similar results with triggers: https://docs.cozodb.org/en/latest/stored.html#triggers
